src/components/Calculator/Digits/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useCalculator } from "../../../modules/context/Calculator/CalculatorContext";
+import Digit from "./Digit";
+
+const digitList = [9, 8, 7, 6, 5, 4, 3, 2, 1, 0];
+
+const Digits = () => {
+  const { calculator, addFirstDigit, addSecondDigit } = useCalculator();
+
+  const handleDigitClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    if (calculator.firstDigits.length < 3 && !calculator.operation) {
+      addFirstDigit(e.currentTarget.value);
+    } else if (calculator.secondDigits.length < 3) {
+      addSecondDigit(e.currentTarget.value);
+    }
+  };
+
+  return (
+    <div className="digits flex">
+      {digitList.map((number) => {
+        return (
+          <Digit key={number} number={number} onClick={handleDigitClick} />
+        );
+      })}
+    </div>
+  );
```

Comment:
이벤트 위임을 적용해도 잘 동작할 것 같아요!

또한 return 문을 생략해서 `{}` 깊이를 줄이면 가독성이 좀 더 올라갈 것 같네요☺

```suggestion
  return (
    <div className="digits flex" onClick={handleDigitClick} >
      {digitList.map((number) => (
          <Digit key={number} number={number} />
        ))}
    </div>
  );
```

Reply:
[45c9418](https://github.com/next-step/react-calculator/pull/27/commits/45c941823d4e894045b0126677f8f09ebe46b472) div 태그에 말씀해주신대로 이벤트 위임을 활용해 보았습니다!
이벤트 위임을 하다보니 e.currentTarget의 경우에는 이벤트가 바인딩된 태그인 `div`태그가 e.currentTarget의 값으로 들어와 e.target으로 실제 이벤트가 일어난 부분만 값으로 받을 수 있도록 변경 해보았습니다!